pkg/frontend/transport/handler.go

```diff
@@ -204,6 +211,13 @@ func (f *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	shouldReportSlowQuery := f.cfg.LogQueriesLongerThan != 0 && queryResponseTime > f.cfg.LogQueriesLongerThan
 	if shouldReportSlowQuery || f.cfg.QueryStatsEnabled {
 		queryString = f.parseRequestQueryString(r, buf)
+		queryDetails := getQueryDetail(r.URL.Path, queryString)
+		stats.AddQuery(queryDetails.QueryString)
+		stats.AddQueryType(string(queryDetails.QueryType))
```

Comment:
Query Type seems redundant to keep into stats, can we remove it?

Reply:
Yeah, removed it.